certified-connectors/PDF4me Connect/apiDefinition.swagger.json

```diff
@@ -1,2496 +0,0 @@
-{
```

Comment:
@anfuca You need to remove other connector files from your PR. But you have removed it from the repo and if your PR gets approved and merged, the other connector swagger would be removed from our repo. Can you please sync your repo with our dev repo and update your PR with only your connector files.